cfg/rke-cis-1.24/master.yaml

```diff
@@ -143,10 +143,12 @@ groups:
         audit: |
           ps -ef | grep $kubeletbin | grep -- --cni-conf-dir | sed 's%.*cni-conf-dir[= ]\([^ ]*\).*%\1%' | xargs -I{} find {} -mindepth 1 | xargs --no-run-if-empty stat -c %U:%G
           find /var/lib/cni/networks -type f 2> /dev/null | xargs --no-run-if-empty stat -c %U:%G
+
         use_multiple_values: true
         tests:
           test_items:
             - flag: "root:root"
+        type: manual
```

Comment:
What's the rational behind setting the type as "manual" for an automated check? Ideally, we should avoid doing this.

Reply:
This was by mistake. The check number I intended to change in was in a different folder. Correct PR here - https://github.com/aquasecurity/kube-bench/pull/1685